### PR TITLE
LAG Doc update for LACP

### PR DESCRIPTION
**Pull request by aashishkuma**
This PR updates the LAG readme for the added support of LACP along with the active-backup mode.

### CODE REVIEW

**Comment by dandaly** [comment]
Can you add to the documentation that active-active can be supported with LACP?

**Review by nupuruttarwar — APPROVED** [review]
LGTM. Please fix markdown lint errors before merging

**Review by 5abeel — APPROVED** [review]
same comment as Nupur. Markdown lint errors need to be fixed. Rest LGTM

File: docs/apps/lnw/es2k/es2k-linux-networking-lag.md
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-linux-networking-lag.md:264]: The text within a code block should be valid code in the specified language. There should be a clear distinction between the commands or instructions to be executed and any text you've added to explain. The rule of thumb is: _the reader should be able to copy the text within the code block and paste it directly into their editor or shell_.

One way of doing this is to move the explanatory text outside the code block.

For static LAG:
```code
ip link add bond0 type bond miimon 100 mode active-backup
```
Another is to make the explanatory text a _comment_:
```suggestion
# For static LAG: ip link add bond0 type bond miimon 100 mode active-backup
```

Given the structure of the information you're presenting, I would suggest making the two explanatory lines comments:
```code
# For static LAG: ip link add bond0 type bond miimon 100 mode active-backup
# For dynamic LAG: ip link add bond0 type bond miimon 100 mode 802.3ad
ip link set <IDPF netdev for VSI 10> down
ip link set <IDPF netdev for VSI 10> master bond0
ip link set <IDPF netdev for VSI 11> down
```
Notice that GitHub includes a "copy" button to make it easier for the reader to use the example directly.

File: docs/apps/lnw/es2k/es2k-linux-networking-lag.md
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-linux-networking-lag.md:298]: Insert an empty line between this line and the first entry in the list.

File: docs/apps/lnw/es2k/es2k-linux-networking-lag.md
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-linux-networking-lag.md:306]: Ditto

File: docs/apps/lnw/es2k/es2k-linux-networking-lag.md
Comment (ffoulkes) [inline docs/apps/lnw/es2k/es2k-linux-networking-lag.md:280]: Ditto

**Review by ffoulkes — CHANGES_REQUESTED** [review]
Please make the indicated changes.

**Review by ffoulkes — APPROVED** [review]
Please make the indicated changes before merging.

**This commit fails the P4CP lint checks / markdownlint** checker. (See the list of checks below. Click on **Details** to see the list of errors.) GitHub will block the merge until you fix the problem.

Note that the checker reports three issues, not two. You need to remove one or more trailing blanks (space characters at the end of the line) from the specified line.

**Pull request merged**
